app/scripts/controllers/transactions/lib/util.js

```diff
@@ -1,4 +1,6 @@
 import { ethErrors } from 'eth-rpc-errors';
+import { ethers } from 'ethers';
```

Comment:
This should be `import { Interface } from '@ethersproject/abi';`

We recently started only importing specific modules. And this will get the lint failure passing

Reply:
Changed in https://github.com/MetaMask/metamask-extension/pull/17326/commits/fd38188dacbed286ec0b39fd6b24ad78daee57d3